booster_bdd/config/config.sh

```diff
@@ -89,6 +89,9 @@ export PIPELINE="${PIPELINE:-maven-releasestageapproveandpromote}"
 ## github repo name
 export GIT_REPO="${GIT_REPO:-test123}"
 
+## github repo URL (for use in creating Che workspace)
+export GIT_REPO_URL="${GIT_REPO:-https://github.com/che-samples/console-java-simple.git}"
+
```

Comment:
The following variable used in `config/prod-preview.sh` is missing here:
```
## URI of the Openshift.io's Che server
export CHE_API="${CHE_API:-https://che.openshift.io}"

```

Reply:
Changes made - many thanks!